spec/migrations/20180606155924_move_ansible_container_secrets_into_database_spec.rb

```diff
@@ -111,7 +117,9 @@
   end
 
   def expect_request
-    expect(described_class).to receive(:read_token).with(token_path).and_return("totally-a-token")
+    token.write("totally-a-token")
+    token.close
```

Comment:
Avoid the expectation on the migration stub constant since it's been removed and re-loaded by the migration.

Reply:
I also like avoiding `expect` all together.

👍